web/client/README.md

```diff
@@ -1,8 +1,12 @@
 # GTFS Web Validator Client
 
+## Node version
+
+Node version: 20+
+
 ## Developing
 
-Once you've created a project and installed dependencies with `npm install` (or `pnpm install` or `yarn`), start a development server:
+Once you've created a project and installed dependencies with `npm install`, start a development server:
```

Comment:
Select npm as the package manager to avoid non-deterministic builds using different package managers concurrently on the repository.